spec/10-expressions.md

```diff
@@ -2982,6 +2984,20 @@ used and each value is `NULL`.
 If the generator function definition declares that it returns byRef,
 each value in a key/value pair is yielded byRef.
 
+The following applies only to the `yield from` form:
+
+A generator function (referred to as a *delegating generator*) can delegate to another generator function (referred to as a *subgenerator*), a Traversable object, or an array, each of which is designated by *expression*.
+
+Each value yielded by *expression* is passed directly to the delegating generator's caller.
+
+Each value sent to the delegating generator's `send` method is passed to the subgenerator's `send` method. If *expression* is not a generator function, any sent values are ignored.
+
+Exceptions thrown by *expression* are propagated up to the delegating generator.
+
+Upon traversable completion, `NULL` is returned to the delegating generator if the traversable is not a generator. If the traversable is a generator, its return value is sent to the delegating generator as the value of the `yield from` *expression*.
+
+An exception of type `EngineException` is thrown if *expression* evaluates to a generator that previously terminated with an uncaught exception, or it evaluates to something that is neither Traversable nor an array.
```

Comment:
Should be `Error` instead of `EngineException` nowadays. Another error condition is doing a `yield from` on the currently running generator.